lib/game.dart

```diff
@@ -23,6 +23,10 @@ abstract class Game {
   // Widget Builder for this Game
   final builder = WidgetBuilder();
 
+  /// Returns the game background color.
+  /// By default it will return a black color.
+  Color backgroundColor() => const Color(0xFF000000);
```

Comment:
Please add to the comments explaining that this value cannot be changed at runtime.

Reply:
Agreed, that can clear up some confusion beforehand. Consider it done